scenarios/generate-synthetic-data/ai-generated-data-with-conversation-starter/README.md

```diff
@@ -0,0 +1,31 @@
+---
+page_type: sample
+languages:
+- python
+products:
+- azure-openai
+description: Use the Simulator to generate high-quality simulated data with pre defined conversation starters using LLMs.
+---
+
+## Generate conversations from conversation starter
+
+### Overview
+
+Large Language Models (LLMs) can help you create question and answer datasets from your existing data sources. These datasets can be useful for various tasks, such as testing your retrieval capabilities, evaluating and improving your RAG workflows, tuning your prompts and more. In this sample, we will explore how to use the Simulator to generate high-quality simulated data with pre defined conversation starters using LLMs.
```

Comment:
Please use the text from the other README. Also please add differentiators between the two readmes for the two different modes as right now these are identical to each other.